css/styles.css

```diff
@@ -160,6 +175,7 @@
 
 .ab-column > ul > li:after,
 .ab-column > ul > li:before {
+    width: 0;
```

Comment:
This part  was restored from 2d193ccc36be60025bd5d8d82f6238b0203b5ffb - but I agree with you, it _seems_ to be without actual function. Crossbrowser-effects should be investigated.